public/app/core/services/util_srv.ts

```diff
@@ -55,50 +51,6 @@ export class UtilSrv {
     this.reactModalRoot.removeChild(this.reactModalNode);
   };
 
-  /**
-   * @deprecated use showModalReact instead that has this capability built in
-   */
-  hideModal() {
-    deprecationWarning('UtilSrv', 'hideModal', 'showModalReact');
-    if (this.modalScope && this.modalScope.dismiss) {
-      this.modalScope.dismiss();
-    }
-  }
-
-  /**
-   * @deprecated use showModalReact instead
-   */
-  showModal(options: any) {
```

Comment:
Removing this needs to be clearly communicated to the plugins communitu, as there may be angular panels that that still use this API. cc @mckn @jackw

Reply:
ah gotcha. didn't think about plugins! so are we even in a position to do the cleanup in this file yet, or do we wait a bit for 9.0 or something? 🤔

Reply:
Sorry @ashharrison90 but given these are breaking changes I think we've gonna need to wait for a major release for this work.

Reply:
that's ok, makes sense 👍  i'll close for now, roll on 9.0! 🚀